### PR TITLE
Add primaries option to `bind_config` resource

### PR DESCRIPTION
**Pull request by detjensrobert**
# Description

This was a missing option from the config file, used to abbreviate lists for `to-notify` and `primaries` in zones.

https://bind9.readthedocs.io/en/v9_16_23/reference.html?highlight=primaries#primaries-statement-definition-and-usage

## Issues Resolved

(none)

## Check List

- [x] A summary of changes made is included in the CHANGELOG under `## Unreleased`
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable.

### CODE REVIEW

File: resources/config.rb [hunk 1]
```diff
@@ -3,49 +3,67 @@
...
           description: 'The desired full path to the configuration file containing options'
+
 property :per_view_additional_config_files, Array,
           default: [],
           description: 'Array of additional per view config files to include in named.conf'
+
 property :statistics_channel, [Hash, Array],
           description: 'Presence turns on the statistics channel'
 
+property :primaries, [Hash],
```
Comment (damacus) [inline resources/config.rb:50]:
```suggestion
property :primaries, Hash,
```

**Review by damacus — DISMISSED** [review]
Just an odd array of a single value to fix and this is good to go 👍🏼

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,9 @@ This file is used to list changes made in each version of the bind cookbook.
 
 ## Unreleased
 
+- add `primaries` option to `bind_config`
+  - see
```
Comment (ramereth) [inline CHANGELOG.md:8]: This looks incomplete.

**Review by ramereth — CHANGES_REQUESTED** [review]

**Review by ramereth — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [3.3.0](https://github.com/sous-chefs/bind/releases/tag/3.3.0)

**Pull request merged**